poucave/utils.py

```diff
@@ -148,7 +148,14 @@ def render_checks(func):
     async def wrapper(request):
         # First, check that client requests supported output format.
         is_text_output = False
-        accepts = set(request.headers.getall("Accept", []))
+        # Several Accept headers can be specified, and each of them can have
+        # comma-separated values (eg. `Accept: text/html,application/xhtml+xml`).
+        accepts = set(
+            chain(*[a.split(",") for a in request.headers.getall("Accept", [])])
```

Comment:
There's also `chain.from_iterable()`, which is meant for this use case. (A pity they gave it that stupid name instead of just calling it `flatten()`.)